tests/stubs/devices/services/temperature.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package services
+
+import (
+	"fmt"
+	"github.com/paypal/gatt"
+)
+
+var dataWrite string
+var write bool
+var state string
+
+func NewTemperatureService() *gatt.Service {
+	temp := 36
+	s := gatt.NewService(gatt.MustParseUUID("09fc95c0-c111-11e3-9904-0002a5d5c51b"))
+
+	// Creating characteristic UUID for reading temperature
+	readCharacteriscticUUID := "11fac9e0-c111-11e3-9246-0002a5d5c51b"
```

Comment:
Can we assign this as a global constant at the top of the file?  It would be good if we can declare all the UUIDs used in the code as constants at one single location

Reply:
Updated